next.config.js

```diff
@@ -67,7 +67,7 @@ const nextConfig = {
               "script-src-attr 'none'",
               `connect-src 'self' ${
                 process.env.NODE_ENV === "development" ? "webpack://*" : ""
-              } https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.ingest.sentry.io`,
+              } https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.ingest.sentry.io https://incoming.telemetry.mozilla.org`,
```

Comment:
This is fine for the a/a test, but we should consider routing through our own proxy eventually (I know this host is on some block lists).

Reply:
@rhelmer yes. I believe we (Data Org) should prioritize providing guidelines for this. Let's work together after the A/A test.